compose.yaml

```diff
@@ -9,16 +9,17 @@ services:
       - 8000:8000
     environment:
       - POP_CONFIG_PATH=${POP_CONFIG_PATH:-/root/.pop/config.yaml}
-      - OLLAMA_HOST=ollama
+      - OLLAMA_HOST=${OLLAMA_HOST:-ollama}
       - OPENAI_API_KEY=${OPENAI_API_KEY}
     volumes:
       - pop:/root/.pop
+      - type: bind
+        source: docker/entrypoint_api.sh
+        target: /root/.pop/entrypoint.sh
```

Comment:
Agree with your remark on bind (should just be a default volume type).

However, on `yaml` configuration form, I prefer using the normal syntax rather than the short form. It is easier to read and therefore to maintain.